justfile

```diff
@@ -2,32 +2,66 @@ set dotenv-load
 set positional-arguments
 
 
-test_dsl:
+test:
   #!/usr/bin/env bash
   set -euxo pipefail
 
-  PGPASSWORD=$CS_DATABASE__PASSWORD dropdb --force --if-exists --username $CS_DATABASE__USERNAME --port $CS_DATABASE__PORT cs_migrator_test
-  PGPASSWORD=$CS_DATABASE__PASSWORD createdb --username $CS_DATABASE__USERNAME --port $CS_DATABASE__PORT cs_migrator_test
+  PGPASSWORD=$CS_DATABASE__PASSWORD dropdb --force --if-exists --username ${CS_DATABASE__USERNAME:-$USER} --port $CS_DATABASE__PORT $CS_DATABASE__NAME
+  PGPASSWORD=$CS_DATABASE__PASSWORD createdb --username ${CS_DATABASE__USERNAME:-$USER} --port $CS_DATABASE__PORT $CS_DATABASE__NAME
 
-  connection_url=postgresql://$CS_DATABASE__USERNAME:@localhost:$CS_DATABASE__PORT/cs_migrator_test
-  PGPASSWORD=$CS_DATABASE__PASSWORD psql $connection_url -f sql/dsl-core.sql
-  PGPASSWORD=$CS_DATABASE__PASSWORD psql $connection_url -f sql/dsl-config-schema.sql
-  PGPASSWORD=$CS_DATABASE__PASSWORD psql $connection_url -f sql/dsl-config-functions.sql
-  PGPASSWORD=$CS_DATABASE__PASSWORD psql $connection_url -f sql/dsl-encryptindex.sql
+  connection_url=postgresql://${CS_DATABASE__USERNAME:-$USER}:@localhost:$CS_DATABASE__PORT/$CS_DATABASE__NAME
+
+  # Install
+  PGPASSWORD=$CS_DATABASE__PASSWORD psql $connection_url -f release/cipherstash-encrypt.sql
 
   # tests
   PGPASSWORD=$CS_DATABASE__PASSWORD psql $connection_url -f tests/core.sql
   PGPASSWORD=$CS_DATABASE__PASSWORD psql $connection_url -f tests/config.sql
   PGPASSWORD=$CS_DATABASE__PASSWORD psql $connection_url -f tests/encryptindex.sql
 
-  dropdb --username $CS_DATABASE__USERNAME --port $CS_DATABASE__PORT cs_migrator_test
+  # Uninstall
+  PGPASSWORD=$CS_DATABASE__PASSWORD psql $connection_url -f release/cipherstash-encrypt-uninstall.sql
 
+  dropdb --username ${CS_DATABASE__USERNAME:-$USER} --port $CS_DATABASE__PORT $CS_DATABASE__NAME
 
 build:
   #!/usr/bin/env bash
   set -euxo pipefail
 
-  cat sql/database-extensions/postgresql/install.sql sql/ore-cllw.sql sql/ste-vec.sql sql/dsl-core.sql sql/dsl-config-schema.sql sql/dsl-config-functions.sql sql/dsl-encryptindex.sql > release/cipherstash-encrypt-dsl.sql
+  mkdir -p release
+
+  rm -f release/cipherstash-encrypt-uninstall.sql
+  rm -f release/cipherstash-encrypt.sql
+
+  # Uninstall
+  # Collect all the drops
+  grep -h '^DROP' sql/0*-*.sql | tac > release/cipherstash-encrypt-uninstall.sql
+  cat sql/600-drop.sql >> release/cipherstash-encrypt-uninstall.sql
```

Comment:
Is there an uninstall for the ore types included anywhere? https://github.com/cipherstash/encrypt-query-language/blob/main/sql/database-extensions/postgresql/uninstall.sql

Reply:
The ore functions have been added as `000-ore.sql`
Includes the various drops.